spikes/npgsql-vanilla/README.rst

```diff
@@ -0,0 +1,40 @@
+#################################################################
+Basic demonstration program for using CrateDB with vanilla Npgsql
+#################################################################
+
+
+************
+Introduction
+************
+
+CrateDB versions prior to 4.2 needed a custom fork of the official Npgsql .NET
+data provider for PostgreSQL, `crate-npgsql`_. CrateDB versions 4.2 and later
+work with the vanilla Npgsql driver without the need for a plugin.
+
+The file ``DemoProgram.cs`` is a demonstration program written in C# which
+outlines typical, very basic usage scenarios with CrateDB.
+ 
+
+*****
+Usage
+*****
+
+In order to invoke a CrateDB instance for evaluation purposes, run::
+
+    docker run -it --rm --publish=4200:4200 --publish=5432:5432 crate:4.5.1
```

Comment:
our standard syntax for console examples is:

```
.. code-block:: console

    sh$ echo foo
```

(here and elsewhere in the file)

Reply:
Thank you!

a) With 5960fa3f, I added `.. highlight: console`, which will essentially do the same job, right?
b) Do you really think we should keep adding prompts like `sh$ ` everywhere?

Reply:
a) yes, great! however, the highlight directive should appear on the first line of the file (for consistency with the rest of our docs)

b) the `sh$` prompt was designed for consistency with the frequent use of `cr>` prompts in the main docs. we can revisit that decision if you like, but I think that should be done separately (perhaps with a PR to crate-docs/style)

Reply:
All right. Addressed with f0cb179c912ce and https://github.com/crate/crate-docs/issues/86.